app/src/main/java/com/github/studydistractor/sdp/StudyDistractorScreen.kt

```diff
@@ -99,17 +99,17 @@ fun StudyDistractorApp(
             composable(route = StudyDistractorScreen.Register.name) {
                 RegisterScreen(
                     onRegistered = {
-                        navController.navigate(StudyDistractorScreen.CreateAccount.name)
+                        navController.navigate(StudyDistractorScreen.CreateUser.name)
                     },
                     registerAuth = FirebaseRegisterAuth(Firebase.auth)
                 )
             }
-            composable(route = StudyDistractorScreen.CreateAccount.name) {
-                CreateAccountScreen(
-                    onAccountCreated = {
+            composable(route = StudyDistractorScreen.CreateUser.name) {
+                CreateUserScreen(
+                    onUserCreated = {
                         navController.navigate(StudyDistractorScreen.Maps.name)
                     },
-                    createAccount = FirebaseCreateAccount()
+                    userService = FirebaseUserService()

```

Comment:
Pass it by depedency injection please, otherwise we can't do end to end testing locally. (Create the object in the main activity and pass it to the screen). I know we did the mistake for historyInterface aswell. (If you want you can fix both at the same time but you don't have to, it is not you task)